### ISSUE TITLE
Transfer link not working with singapore M1 and australia vodafone?

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#1141]
Via @zhangzhongnan928

**Comment by zhangzhongnan928** [comment]
Now, Cannot even load the import page
![8FBF5C8F-55C2-417D-8DBC-E74B8BB7A6F6](https://user-images.githubusercontent.com/33795543/55045325-660df880-5078-11e9-927e-cdcf5a6d502e.png)

**Comment by zhangzhongnan928** [comment]
Australia Vodafone network neither.
This should be the issue from the payment server @colourful-land

**Comment by hboon** [comment]
I tested with a Ropsten transfer link and it worked on M1. Is it reproducible every time on your side for M1 @zhangzhongnan928?

**Comment by zhangzhongnan928** [comment]
Sorry not M1, Circles

		
Victor Zhang
CEO, AlphaWallet
							
RT @mwill_crypto: 3/ 🅰️ @AlexSaundersAU sits down with @Victor928(CEO & Co-Founder, @Alpha_Wallet) to discuss how Alpha Wallet differentia…	Read More

**Comment by hboon** [comment]
Works for me for Circles.Life too.

**Comment by zhangzhongnan928** [comment]
yes, I tested just now, it works on Circles now.
Can you test on Vodafone @James-Sangalli ?

**Comment by hboon** [comment]
Circles.Life is an MVNO leasing from M1. So they probably fails/failed at the same time.

**Comment by zhangzhongnan928** [comment]
I tested last night on circles. It keeps loading for the free transfer link.
On wifi, it shows up import page instantly. 

		
Victor Zhang
CEO, AlphaWallet
							
Not only display information, @Alpha_Wallet let you preform the functions: DEPOSIT, WITHDRAW, PAYBACK, GENERATE, MO… https://t.co/6dJfdvnM3i

**Comment by hboon** [comment]
Just keeping this issue for a bit to remind us ask Victor to confirm that it works for M1/Circles.Life. Not sure it's the same problem as Vodafone's because M1/Circles.Life doesn't consistently fail.

**Comment by zhangzhongnan928** [comment]
Works fine today.
Let me try again tomorrow, then we can close it.

**Comment by James-Sangalli** [comment]
@hboon @zhangzhongnan928 you cannot know this until you get a new build, we changed the routing using elastic load balancer on AWS. This stops the need for use of port 80 as the ELB will reroute all the calls via port 443. This problem should absolutely not exist at all when using the new build, no matter what network you are on.

**Comment by hboon** [comment]
> This problem should absolutely not exist at all when using the new build, no matter what network you are on.

@James-Sangalli yes, hence the need to verify after the build specially for M1/Circles.Life because I'm not sure it's the same problem which is solved by the port change.

**Comment by zhangzhongnan928** [comment]
works fine on Circles now
